tests/client/disco/components/TestAddon.js

```diff
@@ -289,18 +293,29 @@ describe('<Addon />', () => {
     beforeEach(() => {
       previewTheme = sinon.spy();
       resetPreviewTheme = sinon.spy();
-      const data = { ...result, type: ADDON_TYPE_THEME, previewTheme, resetPreviewTheme };
+      const data = {
+        ...result,
+        type: ADDON_TYPE_THEME,
+        previewTheme,
+        resetPreviewTheme,
+      };
       root = renderAddon({ addon: data, ...data });
       themeImage = findDOMNode(root).querySelector('.theme-image');
     });
 
-    it('runs theme preview onMouseOver on theme image', () => {
-      Simulate.mouseOver(themeImage);
+    afterEach(() => {
+      unmountComponentAtNode(findDOMNode(root));
```

Comment:
The out of memory thing is a known issue unfortunately. We're not really sure what causes it.

It might be a good idea to do this in our test setup script. I guess the `componentWillUnmount` callback isn't triggered when the element gets garbage collected?

We could have other components that are leaking memory, but that might not matter much with karma's architecture. I think it's the node part that keeps crashing, I believe each test suite run is in an iframe that gets removed after completion.